inspirehep/modules/theme/views.py

```diff
@@ -145,6 +149,53 @@ def data():
 def ping():
     return 'OK'
 
+#
+# Handlers for AJAX requests regarding references and citations
+#
+
+
+@blueprint.route('/ajax/references', methods=['GET'])
+def ajax_references():
+    """Handler for datatables references view"""
+
+    if request.args.get('recid'):
+        recid = request.args.get('recid', '')
+        collection = request.args.get('collection', '')
+
+    from flask import jsonify
+    from inspirehep.utils.references import Reference
+    from invenio_pidstore.models import PersistentIdentifier
+
+    pid = PersistentIdentifier.get(collection, recid)
+
+    record = current_search_client.get_source(index='records-hep',
+                                              id=pid.object_uuid,
+                                              doc_type='hep',
+                                              ignore=404)
+
+    return jsonify(
+        {
+            "data": Reference(record).references()
+        }
+    )
+
+
+@blueprint.route('/ajax/citations', methods=['GET'])
+def ajax_citations():
+    """Handler for datatables citations view"""
+    # recid = request.form.get('recid')
```

Comment:
Seems to be ported here? https://github.com/inspirehep/inspire-next/pull/944/files#diff-cdf08c45f59c56a33c8696da6d4478eaR46